### DISCUSSION TITLE
评论中关于emoji表情的小问题？

### DISCUSSION THREAD

**Discussion by suyinLoong** [walinejs/waline#195] [Q&A] [answered]
![QQ截图20210408102658](https://user-images.githubusercontent.com/49554285/113959171-faf58000-9854-11eb-9c48-d37a1911e54d.png)

**Comment by lizheming** [comment]
这应该是你的主题的样式导致的样式冲突，建议你样式覆盖处理一下。

**Reply by suyinLoong** [reply]
> 这应该是你的主题的样式导致的样式冲突，建议你样式覆盖处理一下。

好的，谢谢！

**Comment by suyinLoong** [comment]
我用的是Next主题，使用的评论插件是[@waline/hexo-next](https://github.com/lizheming/waline/tree/master/packages/hexo-next)
希望可以针对该主题优化一下这个emoji表情的显示问题，谢谢！

**Comment by suyinLoong** [comment, marked as answer]
目前我是通过修改Next自定义文件来解决的该问题。
具体步骤如下：

1. 在博客目 `\source\_data\` 下新建`styles.styl`文件，如：`\Hexo\source\_data\styles.styl`
2. 然后在`styles.styl`中写入以下内容
    ```styl
    .v[data-class=v] img {
        display: inline;
    }
    ```
3. 再然后在Next主题配置文件中打开 `custom_file_path:` 下的 `style: source/_data/styles.styl` 选项
4. 最后运行`hexo clean && hexo g && hexo s`重新生成静态文件，然后部署即可。

**Comment by ReaJason** [comment]
请问怎么让这个评论后显示的表情和表情选择框中展示的一样大，我添加了billbill表情，显示很大，图片锯齿很吓人
![Screenshot_2021-04-11-11-38-24-281_com microsoft emmx](https://user-images.githubusercontent.com/48389642/114291501-74da7300-9aba-11eb-9f5c-df022f0e242c.jpg)
![Screenshot_2021-04-11-11-38-12-684_com microsoft emmx](https://user-images.githubusercontent.com/48389642/114291503-76a43680-9aba-11eb-81f4-2d31eae5716f.jpg)

**Reply by lizheming** [reply]
在主题里自己补一下 CSS 重新设置下表情的大小吧

```css
.v[data-class=v] img {
    width: 1.5em
}
```

**Reply by ReaJason** [reply]
谢谢解答🥰
